qcoptions.py

```diff
@@ -8,7 +8,7 @@
                   help="Minimum value of scan parameter", metavar="scanmin", default=0)
 parser.add_option("--scanmax", type="int", dest="scanmax",
                   help="Maximum value of scan parameter", metavar="scanmax", default=254)
-parser.add_option("--vfatmask", type="int", dest="vfatmask",
+parser.add_option("--vfatmask", type="int", dest="vfatmask", 
```

Comment:
Mmhmm, maybe It's something in my .vimrc then...from terminal in my editor I do not see these extra spaces but they do appear:

https://github.com/bdorney/vfatqc-python-scripts/blob/master/qcoptions.py

If I highlight the text in github...